script/build-task.js

```diff
@@ -70,6 +70,11 @@ export const buildTask = async (
 				},
 				*/
 				// endregion
+				{
+					id: "lib-wrapper",
+					type: "module",
+					manifest: "https://github.com/ruipin/fvtt-lib-wrapper/releases/latest/download//module.json",
```

Comment:
double `/` in the manifest link. 

```suggestion
					manifest: "https://github.com/ruipin/fvtt-lib-wrapper/releases/latest/download/module.json",
```